app/components/pull-request-page.js

```diff
@@ -67,6 +67,19 @@ export default Ember.Component.extend({
     )
   },
 
+  visible: Ember.computed("model.status", "session.isCommitter", function() {
+    let committer = get(this, "session.isCommitter")
+    let status = get(this, "model.status")
+
+    if ((status === "draft" || "revise-and-resubmit") && committer === null) {
```

Comment:
instead of null, I'd do the opposite of true which is what you test for in line 76. It would be easy to set "isCommitter" in the database to false, assuming that would work. By testing for null instead of "not true", it would break. Also, by testing for the opposite in two different branches make it's easier to read and reduces the branching factor as there are fewer possibilities)